lib/chef/knife/winrm_knife_base.rb

```diff
@@ -122,11 +122,12 @@ def relay_winrm_command(command)
 
             queue = Queue.new
             @winrm_sessions.each { |s| queue << s }
+            number_of_session = config[:concurrency] ? locate_config_value(:concurrency) : @name_args[0].split(" ").count || queue.size
```

Comment:
`@name_args[0].split(" ").count || queue.size`
As count would always return a number, `queue.size` would be non-reachable here.
Could you please confirm if we need a `zero?` check with the count or may be an empty check with the array here.